### DISCUSSION TITLE
Package OpenRCT2 data (e.g. g.dat etc) with Android release

### DISCUSSION THREAD

**Discussion by adammw** [OpenRCT2/OpenRCT2#19118] [Feature requests]
I just updated my Android release and had problems with the previous data as it is strictly tied to the game version.

Given all the other platforms embed this data, the Android release should as well to make upgrades seamless.

**Comment by janisozaur** [comment]
I started work on that a lot of time ago and have to revive it. I should have a bit more time in a few weeks for that. I don't recall if there is already pr or branch available.
